pep-9999.rst

```diff
@@ -239,8 +237,8 @@ argument of the ``__init__`` method::
         def __init__(self, type: str) -> None: ...
 
 The class must match the class in which it is declared. Using other classes,
-including sub or super classes will not work. In addition, the ``self``
-annotation can not contain type variable itself.
+including sub or super classes, will not work. In addition, the ``self``
+annotation can not contain the type variable itself.
```

Comment:
```suggestion
annotation cannot contain the type variable itself.
```

Reply:
Thanks! I also found and fixed one more "can not".